tests/pyreverse/data/classes_No_Name.html

```diff
@@ -2,7 +2,7 @@
   <body>
     <script src="https://cdn.jsdelivr.net/npm/mermaid/dist/mermaid.min.js"></script>
       <div class="mermaid">
-
+    
```

Comment:
Just a minor nitpick: can you remove the whitespace here and in line 50 please?

Reply:
Good catch, removed in commit c6cea8088cc38ee6fbb1eef6077b2bad6f3503cd